### PR TITLE
Fix bug in Arithmetic unary op implementation

### PR DESCRIPTION
**Pull request by jantonguirao**
## Category:

**Bug fix**

## Description:

If fixes a bug in math/arithmetic unary operators GPU implementation: The tile offset was added twice.

### Affected modules and functionalities:

Arithm ops

### Key points relevant for the review:

Bugfix, test coverage

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [x] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [ ] N/A
test_arithmetic_ops.py:
- test_math_function_ops_big
- test_unary_arithmetic_ops_big

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: DALI-3344

### CODE REVIEW

File: dali/test/python/operator_1/test_arithmetic_ops.py [hunk 1]
```diff
@@ -452,6 +452,22 @@ def test_unary_arithmetic_ops():
...
+
+def test_ternary_ops_big():
+    for kinds in selected_ternary_input_kinds:
+        for (op, op_desc) in ternary_operations:
+            for types_in in [
+                (np.int32, np.int32, np.int32),
+                (np.int32, np.int8, np.int16),
+                (np.int32, np.uint8, np.float32),
+            ]:
+                yield check_ternary_op, kinds, types_in, op, shape_big, op_desc
```
Comment (klecki) [inline dali/test/python/operator_1/test_arithmetic_ops.py:468]: Copy-paste of existing test.

**Review by klecki — APPROVED** [review]
> If fixes a bug in math/arithmetic unary operators GPU implementation

It would be good to say what the error is, just for the record, eg: "the tile offset was added twice".

**Review by stiepan — APPROVED** [review]

File: dali/test/python/operator_1/test_arithmetic_ops.py [hunk 2]
```diff
@@ -452,6 +452,22 @@
                 if types_in != np.bool_:
                     yield check_unary_op, kinds, types_in, op, shape_small, op_desc
 
+def test_unary_arithmetic_ops_big():
+    for kinds in unary_input_kinds:
+        for (op, op_desc) in unary_operations:
+            yield check_unary_op, kinds, np.int8, op, shape_big, op_desc
+
+def test_ternary_ops_big():
```
[the remark this answers is not in the document]
Comment (stiepan) [reply dali/test/python/operator_1/test_arithmetic_ops.py:460]: It has a point, the test with such a name already exists (line 552).

[the remark this answers is not in the document]
Comment (jantonguirao) [reply dali/test/python/operator_1/test_arithmetic_ops.py:460]: Of course, leftover copy-paste

**Comment by jantonguirao** [comment]
!build

**Review by stiepan — APPROVED** [review]

**Comment by jantonguirao** [comment]
!build

**Pull request merged**